app/internal/page/app.go

```diff
@@ -55,12 +39,17 @@ func (l Lang) String() string {
 	return englishAbbreviation
 }
 
-func (l Lang) BuildUrl(url string) string {
-	if l == Cy {
-		return "/" + welshAbbreviation + url
-	} else {
-		return url
-	}
+func CacheControlHeaders(h http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Cache-Control", "max-age=2592000")
+		h.ServeHTTP(w, r)
+	})
+}
+
+func IsLpaPath(url string) bool {
+	path, _, _ := strings.Cut(url, "?")
+
+	return path != Paths.Dashboard
 }
```

Comment:
This isn't great, but it does solve the problem for now. I'm thinking eventually it would be nice to make the path definitions as structs so we can attach a bit more info to them, and make them self formatting (like `Paths.YourDetails.Format(123) == "/lpa/123/your-details"` maybe)

Reply:
That sounds like a good idea and this is totes fine for now 👍